python/dataproc_templates/hive/util/README.md

```diff
@@ -0,0 +1,55 @@
+# Hive DDL Extractor
+
+This is a PySpark utility to extract DDLs from Hive metastore. Users can use this utility to accelerate the Hive migration journey.
```

Comment:
Explicitly say that it does not convert Hive DDLs to BigQuery DDLs automatically. That it dumps the result to GCS and metadata to BigQuery.
The advantage of this utility template is to get DDLs from Hive Metastore databases in a scalable manner, handling large Hive Metastores